portal/src/error/parse.ts

```diff
@@ -91,13 +95,53 @@ function isKnownCauseKind(kind: string): boolean {
   return kind in errorCauseMessageIDs;
 }
 
+function getFormatErrorMessage(cause: FormatErrorCause): ParsedAPIError | null {
```

Comment:
@louischan-oursky 

For this error, actually it applies to all duration string in config. So I think can keep here instead of moving to component rules. What do you think?

Reply:
OK.